.evergreen.yml

```diff
@@ -77,9 +91,40 @@ tasks:
       - func: "gen-notices"
       - func: "fetch goreleaser"
       - func: "package"
+      - command: shell.exec
+        params:
+          working_dir: mongocli/dist
+          script: |
+            set -o errexit
+            set -o verbose
+
+            # we could generate a similar name with goreleaser but we want to keep the vars evg compatibale to use later
+            mv mongocli_next_linux_i386.deb mongocli_next_${revision}_${version_id}_${created_at}_linux_i386.deb
```

Comment:
I assume EVG populates `version_id` and `created_at`?

Reply:
yes, those are part of the default expansions, https://github.com/evergreen-ci/evergreen/wiki/Project-Files#expansions